typing-c/src/C/Typing/Clang.hs

```diff
@@ -0,0 +1,315 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module C.Typing.Clang
```

Comment:
Does this need to be in the library component of `typing-c`, or could it just be in the testsuite?